arm/Microsoft.ServiceFabric/clusters/.bicep/nested_applications.bicep

```diff
@@ -0,0 +1,24 @@
+param applicationObj object
```

Comment:
This folder should be moved into its own child resource folder. See upstream storage account for reference

Reply:
Moved referencing upstream.